tests/pallas/tpu/ops_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for common JAX operations within pallas_call."""
+
+import numpy as np
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import jax
+import jax.numpy as jnp
+from jax import lax
+from jax._src import test_util as jtu
+from jax.experimental import pallas as pl
+
+# Import mosaic for flag definitions
+from jax.experimental import mosaic as _  # noqa: F401
+
+
+class OpsTest(jtu.JaxTestCase):
```

Comment:
Shall we re-use pallas/ops_test.py?

Reply:
Ah yeah good call, didn't see we had that. Though will this pass on GPU?

Reply:
It should, since it's a lower_fun rule?